tools/clang/lib/SPIRV/DeclResultIdMapper.h

```diff
@@ -300,13 +300,14 @@ class DeclResultIdMapper {
 
   /// \brief Creates a function-scope variable in the current function and
   /// returns its <result-id>.
-  uint32_t createFnVar(const VarDecl *var, llvm::Optional<uint32_t> init);
+  SpirvEvalInfo createFnVar(const VarDecl *var, llvm::Optional<uint32_t> init);
```

Comment:
It's not super clear to me. But do we need to change the signature of these `create*Var` methods?

Reply:
discussed offline.